### PR TITLE
EN-927 user list component, state management and unit tests created

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by mcarta-rt** [comment]
Missing user profile info. Waiting for User / Profile API definition.

**Comment by entando-service** [comment]
All is well

File: src/locales/it.js [hunk 1]
```diff
@@ -156,5 +158,15 @@ export default {
...
     'dataType.table.status.wip': 'Ricaricamento in corso. Aggiorna la pagina.',
     'dataType.listEmpty': 'Non ci sono DATA TYPES disponibili',
+    'user.list.title': 'Utenti',
+    'user.help': 'Dalla sezione UTENTI l\'amministratore può accedere alla lista degli utenti, gestirne i profili, le autorizzazioni - assegnando ruoli e gruppi - e aggiungere nuovi utenti.',
+    'user.table.status.active': 'Attivo',
+    'user.table.status.disabled': 'Non attivo',
+    'user.table.username': 'Username',
+    'user.table.status': 'Stato',
+    'user.listEmpty': 'Non ci sono UTENTI disponibili',
+    'user.action.manageAuth': 'Manage autorization for: {username}',
```
Comment (alessioscalici) [inline src/locales/it.js]: Looks like these 3 labels are not in italian

File: src/ui/users/list/UserListMenuActions.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    return (ev) => {
+      ev.preventDefault();
+      if (handler) { handler(this.props); }
+    };
+  }
+
+  render() {
+    const { onClickDelete } = this.props;
+    const editLabel = (
+      <FormattedMessage id="app.edit" values={{ username: this.props.username }} />
```
Comment (alessioscalici) [inline src/ui/users/list/UserListMenuActions.js]: app.edit message has no "username" parameters

File: src/ui/users/list/UserListMenuActions.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    );
+    const manageAuthLabel = (
```
Comment (alessioscalici) [inline src/ui/users/list/UserListMenuActions.js]: Why declaring const if not reusing them? Please inline these messages

File: src/ui/users/list/UserListMenuActions.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    );
+    const viewProfileLabel = (
+      <FormattedMessage id="user.action.viewProfile" values={{ username: this.props.username }} />
+    );
+    return (
+      <DropdownKebab pullRight id={`${this.props.username}-actions`}>
+
+        <MenuItem
+          className="UserListMenuAction__menu-item-edit"
+          onClick={this.handleClick()}
```
Comment (alessioscalici) [inline src/ui/users/list/UserListMenuActions.js]: Please remove there empty handlers and teh handleClick method: preventDefault should be done automatically by MenuItem component

File: src/ui/users/list/UserListTable.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    this.props.onWillMount(page);
+  }
+
+  renderTableRows() {
+    return this.props.users.map(user => (
+      <tr key={user.username}>
+        <td className="UserListRow__td">{user.username}</td>
+        {/*
+          FIXME: user profiles info
+          <td className="UserListRow__td">{user.fullName}</td>
```
Comment (alessioscalici) [inline src/ui/users/list/UserListTable.js]: Please decomment and place mock (hardcoded) data, to have at least the visual implementation

File: test/state/data-types/selectors.test.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  pagination: DATA_TYPES_OK_PAGE_1.metaData,
+};
+
+
+describe('state/users/selectors', () => {
+  it('getDataTypes(state) returns the users object', () => {
+    const selected = getDataTypes(TEST_STATE);
+    expect(selected).toBe(TEST_STATE.dataTypes);
+  });
+  it('verify getDataTypesIdList selector', () => {
```
Comment (alessioscalici) [inline test/state/data-types/selectors.test.js]: Please :) put an empty line between it blocks

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**